packages/admin/app/components/DisplayAsset.jsx

```diff
@@ -0,0 +1,40 @@
+import styles from '../styles/dist/DisplayItem.css'
+import { useLoaderData } from '@remix-run/react'
+
+function DisplayAsset() {
+  const { asset } = useLoaderData()
```

Comment:
@cairin I'll leave the DisplayAsset and DisplayPeer files as components because my idea is to reuse them as results for the general search functionality as well where we might get a mix of peers/assets depending on what is being searched (e.g asset code). But I'll incorporate the others into their corresponding route files.

Reply:
Sounds good :+1: